### PR TITLE
add write/speak/code to conferences

### PR DESCRIPTION
**Pull request by agonzalez0515**
Relates to #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,8 +217,12 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
     WITI is the premier global organization helping tech-savvy women attain their professional goals. WITI organizes events and summits for women.
 
   - [Women Who Tech](http://womenwhotech.com/)
-
+  
     Organizers of the yearly Women Who Tech TeleSummit, with talks by women in technology, startups, and social media. Creators of the #Women2Follow hashtag on Twitter.
+    
+  - [Write/Speak/Code](http://www.writespeakcode.com/)
+  
+     Write/Speak/Code is on a mission to empower women software developers to become thought leaders, conference speakers, and open source contributors. We run conferences and events across the US.
```
Comment (harsh183) [inline README.md]: Can 'We run...' be changed to something like 'they run...'?

**Comment by agonzalez0515** [comment]
Changed! 😃

**Comment by harsh183** [comment]
Looks great. I think it's ready to merge.

**Comment by cristianoliveira** [comment]
Hey @agonzalez0515 Thanks for your contributions!

As @harsh183 said, looks great! Merging...

**Pull request merged**